src/rb3/unk_8037A49C.cpp

```diff
@@ -0,0 +1,9 @@
+#include "string.hpp"
+#include "textfile.hpp"
+#include "textstream.hpp"
+#include "unknown.hpp"
+
+// fn_8037A67C
+TextStream::TextStream(){ }
+
+// fn_8000DD10
```

Comment:
what's going on here.

is this file not close enough to the rest of the textstream functions to justify making it a single split